kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorIT.java

```diff
@@ -375,8 +375,11 @@ private void assertContinuumMessagesAsExpected(ConsumerRecords<byte[], byte[]> c
         assertTrue(expectedFileNamesCopy.contains(message.filename));
         expectedFileNamesCopy.remove(message.filename); // ensures filenames are distinct
 
-        assertEquals(offset, message.offset);
-        offset += expectedRecordsPerFile;
+        assertEquals(startOffset, message.startOffset);
+        // This is just a loose >= check because it's possible for numbers to get
```

Comment:
I might be misremembering, but I believe this is a true statement.

Reply:
Seems reasonable to me. We can probably copy whatever the tests related to metadata tagging does, if there are any.

Reply:
[Those tests](https://github.com/ni/kafka-connect-storage-cloud/blob/722285da20f63c3e4f2eacfd80c6c9657a3c7a0f/kafka-connect-s3/src/test/java/io/confluent/connect/s3/TopicPartitionWriterTest.java#L1118) are unit tests, and they do a strict equality check on the expected offsets. I think I prefer the looser check here, since this is an integration test.